app/src/main/kotlin/com/wire/android/ui/calling/SharedCallingViewModel.kt

```diff
@@ -185,28 +189,32 @@ class SharedCallingViewModel @Inject constructor(
 
     fun toggleMute() {
         viewModelScope.launch {
-            callState = if (callState.isMuted) {
-                callState.copy(isMuted = false)
-            } else {
-                callState.copy(isMuted = true)
+            callState.isMuted?.let {
+                callState = if (it) {
+                    callState.copy(isMuted = false)
+                } else {
+                    callState.copy(isMuted = true)
+                }
             }
         }
     }
 
     fun toggleVideo() {
         viewModelScope.launch {
-            callState = if (callState.isCameraOn) {
-                turnLoudSpeakerOff()
-                callState.copy(
-                    isCameraOn = false,
-                    isSpeakerOn = false
-                )
-            } else {
-                turnLoudSpeakerOn()
-                callState.copy(
-                    isCameraOn = true,
-                    isSpeakerOn = true
-                )
+            callState.isCameraOn?.let {
```

Comment:
if the default value is `null` would there be any edge case that it will still remain `null` and we will not be able to toggle the camera button?